pages/dkp/konvoy/2.3/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -43,54 +43,60 @@ This section describes how to upgrade your Kommander Management cluster and all
 
   <p class="message--important"><strong>IMPORTANT:</strong> Beginning with DKP version 2.3, MetalLB is no longer managed as a platform application. If you installed  MetalLB on the cluster that you're upgrading prior to DKP version 2.3, you will need to detach MetalLB from the cluster prior to upgrading.</p>
 
-  1. Pause the helm release.
```

Comment:
@takirala, added the changes you made to 2.3 Konvoy docs as well. I know these are not live yet and the process will probably change, but I wanted to make sure we included changes here just in case. 
This section seemed to be up-to-date already and all I did was some linting. 
Can you please check everything we need is here as well?